.mci.yml

```diff
@@ -172,6 +173,7 @@ functions:
                     if [ "Windows_NT" == "$OS" ]; then
                         PREFIX=$(cygpath -m "$PREFIX")
                     fi
+                    export CMAKE=${cmake}
```

Comment:
Does this line have an effect?  Who consumes the new CMAKE variable?  Also, what is the output of ${cmake} ?  When I run that locally in my shell, the result is empty.

Reply:
It overrides the checks in `find_cmake.sh`. A lot of files use that script to find `cmake` iff the env var `CMAKE` isn't set. I set it here to avoid that.